api/v1alpha4/cluster_webhook_test.go

```diff
@@ -43,6 +44,20 @@ func TestClusterDefault(t *testing.T) {
 	g.Expect(c.Spec.ControlPlaneRef.Namespace).To(Equal(c.Namespace))
 }
 
+func TestClusterDefaultingValidation(t *testing.T) {
+	c := &Cluster{
```

Comment:
I don't love how we currently essentially have the same object in several test cases (see `TestClusterDefault` in this case). It would be nice to make this a bit more DRY - maybe something for a follow up.

Reply:
Deduped it in place.